puppeteer/spec/request-modification.js

```diff
@@ -0,0 +1,104 @@
+const puppeteer = require("puppeteer");
+const userListSelector = "[data-id='users']";
+const {
+  BASE_URL,
+  USER_LIST_PATH,
+  SINGLE_USER_PATH,
+} = require("../data/constants.json");
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+const chaiDeepMatch = require("chai-deep-match");
+const expect = chai.expect;
+const expectedResponse = require("../data/response.json");
+chai.use(chaiAsPromised);
+chai.use(chaiDeepMatch);
+chai.use(require("chai-json-schema"));
```

Comment:
I would suggest centralizing the `chai` configuration. This way, you need it in all of your specs.

Reply:
Based on our offline discussion, we can use [ROOT HOOK PLUGINS](https://mochajs.org/#root-hook-plugins)